### ISSUE TITLE
Reading area jumps when menu side panel is open

### ISSUE THREAD

**Issue by sisou** [leggett/simplify#563]
When the main menu left side panel is open, there's a visible jump when switching between email list and reading view:
![ezgif-6-96ea15c56cfc](https://user-images.githubusercontent.com/1828163/111708775-c0e12180-8846-11eb-9618-cb7667ba069d.gif)

---
Simplify v.2.3.7 - Configuration: aAX ca4 caE caI caL cfC cfO cfQ cfR cfU composeActions dateGroup defaultTheme inInbox inList invertAddons invertCompose izBgDefault lightTheme listWidthLg lowDensity matchFontSize minimizeChat msgWidthSm navOpen newUI simplify - System: Mozilla/5.0 (X11 Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36 - Window: 1920 x 1002 (100pct zoom) - Language: en

**Comment by leggett** [comment]
Thanks for the report! The gif + system setup should make this easy to troubleshoot.

**Comment by leggett** [comment]
Ok... I remember what's going on here and why it is this way. 

- Centering the list and conversation in the space available (100% - width of nav) doesn't quite look right as the nav doesn't look like it is as wide as it is.
- I add some right padding on the conversation to compensate for this but not on the list (as it was harder to apply to the list without breaking things -- you have to 
- The end result is the jump you're seeing with the nav is open :(

Bottom line: 

- I half fixed the centering of content with the nav open
- I should probably remove my right padding on the conversation OR figure out how to add it to the list correctly
